sdk/search/azure-search-documents/pom.xml

```diff
@@ -24,7 +24,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <!-- Coverage used to be higher but between additional, unused generated client APIs and unused models' JSON serialization it has been lowered. -->
     <!-- In the future this will be cleaned up and coverage should be increased again. -->
-    <jacoco.min.linecoverage>0.59</jacoco.min.linecoverage>
+    <jacoco.min.linecoverage>0.58</jacoco.min.linecoverage>
```

Comment:
Why does the coverage drop when building with Java 21?

Reply:
I think this change came from merging `main` into this branch. I don't remember changing this manually. I'll make sure I'm not changing it compared to `main`.

Reply:
Reverted this.